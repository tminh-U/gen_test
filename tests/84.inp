8
18 16 3 17 17 20 11 4 
9
5 7
2 6
2 5
4 8
3 3
2 7
7 8
6 6
8 8
