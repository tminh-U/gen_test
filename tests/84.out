5
24
12
10
2
26
3
2
2
